src/locker/idle-counter.js

```diff
@@ -0,0 +1,71 @@
+import signal from 'pico-signals';
+
+const STORAGE_MAXTIME_KEY = 'locker.idle.maxTime';
+const DEFAULT_MAX_TIME = 3 * 60 * 1000;
+
+class IdleCounter {
```

Comment:
Didn't you want to change this name to something else?

Reply:
Yes, we still need to agree on a name. What do you think about IdleTimer?

Reply:
I like it!